docs/installation-places.md

```diff
@@ -1,35 +0,0 @@
-# Installation
```

Comment:
Where did the installation docs for places get moved to or did they maybe get lost in the shuffle?

These should still be included since places is distributed as its own library (AAR) but perhaps we could combine these with the SDK install docs into a single `installation.md`?